netdata.xcodeproj/project.pbxproj

```diff
@@ -1322,7 +1322,7 @@
 				CODE_SIGN_STYLE = Automatic;
 				CURRENT_PROJECT_VERSION = 14;
 				DEVELOPMENT_ASSET_PATHS = "\"netdata/Preview Content\"";
-				DEVELOPMENT_TEAM = H78GYE72WQ;
```

Comment:
This might be a dumb question as I'm new to iOS development, but is there a way to configure this locally?